Doc/library/sqlite3.rst

```diff
@@ -304,8 +304,8 @@ Module functions
    :param bool check_same_thread:
        If ``True`` (default), only the creating thread may use the connection.
        If ``False``, the connection may be shared across multiple threads;
-       if so, write operations should be serialized by the user to avoid data
-       corruption.
+       users should check :attr:`threadsafety` before setting this value to
+       avoid data corruption.
```

Comment:
I worry that the suggested text may come across as too vague. Parallell programming is hard to get right, even for pros. I'd be more comfortable if we keep the current serialisation recommendation, and instead _add_ a sentence where we point to the threadsafety attribute.

How about this:

```suggestion
       if so, write operations may need to be serialized by the user,
       to avoid data corruption.
       Please see :attr:`threadsafety` for more information.
```